deploy/Chart/values.yaml

```diff
@@ -112,3 +112,15 @@ dashboard:
       memory: "60Mi"
     limits:
       memory: "300Mi"
+
+database:
+  image: ghcr.io/radius-project/mirror/postgres
```

Comment:
Would like to hear your thoughts on the [Cloud Native Postgres](https://github.com/cloudnative-pg/cloudnative-pg).

Are we going to install Postgres all the time or only if the user chooses Postgres as the data store?

cc/ @rynowak

Reply:
We are not taking on dependencies like Cloud Native Postgres. Our message to users is that they can host the database wherever/however they like.

It's a **non-goal** for us to provide production support for Postgres out of the box.